client/state/comments/actions.js

```diff
@@ -221,6 +222,13 @@ export function writeComment( commentText, siteId, postId, parentCommentId ) {
 				skipSort
 			} );
 
+			dispatch( {
+				type: COMMENTS_COUNT_INCREMENT,
+				siteId,
+				postId,
+				incrementBy: 1
```

Comment:
What if somebody else added a comment in the meantime? What is our ultimate source of truth?